Road/notebooks/feature_selected.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>Same comment regarding following PEP8 convention in ordering libraries. Could you please adjust it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>